src/main/java/it/pagopa/pn/downtime/controller/EventController.java

```diff
@@ -73,6 +75,9 @@ public ResponseEntity<Void> addStatusChangeEvent(String xPagopaPnUid, List<PnSta
 		PnAuditLogEvent logEvent = auditLogBuilder.before(PnAuditLogEventType.AUD_NT_DOWNTIME,
 				"addStatusChangeEvent - xPagopaPnUid={}, pnStatusUpdateEvent={}", xPagopaPnUid, pnStatusUpdateEvent)
 				.mdcEntry("uid", xPagopaPnUid).build();
+		
+		log.info("AUD_NT_DOWNTIME - Current date (GMT/UTC) = {} " + OffsetDateTime.now());
```

Comment:
Per i log di Audit c'è una classe apposita in pn-commons https://github.com/pagopa/pn-commons/tree/develop/src/main/java/it/pagopa/pn/commons/log che permette di indicizzare questi log in modo puntuale.

Reply:
Sono log aggiunti per il troubleshooting, in quanto stiamo avendo problemi sulle date. Poi questi log verranno rimossi. Comunque l'ho rimosso dal Controller